Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:23.04
+
+# Copies your code file from your action repository to the filesystem path `/` of the container
+COPY entrypoint.sh /entrypoint.sh
+
+# Copy all trivy report data
+COPY *.json /data/
+
+# Mount data volume
+VOLUME /data
+
+# Install required packages
+RUN apt-get update && \
+    apt-get install -y wget tar runc sudo systemd apt-transport-https ca-certificates gnupg lsb-release --no-install-recommends
+
+# Install Copa
+RUN wget -q https://github.com/project-copacetic/copacetic/releases/download/v0.2.0/copa_0.2.0_linux_amd64.tar.gz && \
```

Comment:
add retries to wget unless it has it enabled by default

Reply:
I think the default is 20 times